litellm/proxy/_types.py

```diff
@@ -77,6 +77,15 @@ class LiteLLM_UpperboundKeyGenerateParams(LiteLLMBase):
 
 
 class LiteLLMRoutes(enum.Enum):
+    openai_route_names: List = [
+        "chat_completion",
+        "completion",
+        "embeddings",
+        "image_generation",
+        "audio_transcriptions",
+        "moderations",
+        "model_list",  # OpenAI /v1/models route
```

Comment:
i believe your test will fail for `/v1/models` since the endpoint name is `models` not `model_list`.

Can you sanity check this for me? @msabramo

Reply:
Isn't it `model_list`?

```python
#### API ENDPOINTS ####
@router.get(
    "/v1/models", dependencies=[Depends(user_api_key_auth)], tags=["model management"]
)
@router.get(
    "/models", dependencies=[Depends(user_api_key_auth)], tags=["model management"]
)  # if project requires model list
def model_list(
    user_api_key_dict: UserAPIKeyAuth = Depends(user_api_key_auth),
):
```

Reply:
the function is - maybe i misread the code, i thought the name checked the url 

that's on me - no worries then. Thanks for clarifying.

Reply:
No problem. I'm about to add a few more routes to the test.

Reply:
cf7185735